src/test/java/net/datafaker/base/PhoneNumberTest.java

```diff
@@ -126,6 +126,38 @@ void testAllCellPhone_enPh() throws NumberParseException {
         assertThat(util.isValidNumberForRegion(proto, "PH")).as(phoneNumber).isTrue();
     }
 
+    @Test
+    void testPhoneNumberInternational_US() throws NumberParseException {
+        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
+        String phoneNumber = new BaseFaker(new Locale("en_US")).phoneNumber().phoneNumberInternational();
```

Comment:
I think we prefer to use `Locale("en", "US")`

Reply:
Fixed